packages/create-rise/src/cli.ts

```diff
@@ -77,14 +90,29 @@ async function createRise() {
     }
   )
 
-  cd(root)
+  if (installDeps) {
+    // tbd: offer an option to choose the package manager via options
+    // maybe this is something we can share with the Community CLI?
+
+    cd(root)
 
-  // tbd: offer an option to choose the package manager via options
-  // maybe this is something we can share with the Community CLI?
-  await spinner(`Installing dependencies in ${projectName}`, () => $`npm install`)
+    console.log(chalk.bold(`📦 Using ${chalk.bold.cyan('npm')} to install packages.`))
+
+    await spinner(chalk.bold(`🔧 Installing dependencies in ${projectName}`), () => $`npm install`)
+  }
+
+  console.log(riseGradient(WELCOME_TO_RISE_ASCII))
 
   console.log(
-    `The project has been successfully created in ${projectName}. To start, run 'npm dev'`
+    [
```

Comment:
nit: we can use `decent` and multiline string, e.g.
```
welcome

to

rise
```
instead of the array notation!

Reply:
You meant `dedent` ? I've tried it but didn't know how to conditionally display `installDeps` without making empty line when the value is not there.